discovery-infra/start_discovery.py

```diff
@@ -446,6 +458,28 @@ def set_hosts_roles(client, cluster, nodes_details, machine_net, tf, master_coun
                        update_roles=master_count > 1)
 
 
+def parse_static_ips_config(config_file):
+    with open(config_file) as _file:
+        res = json.load(_file)
+        return res
+
+
+def set_hostames_for_static_ips_hosts(client, cluster_id):
```

Comment:
we have this logic for ipv6 , please use it

Reply:
we don't really need specific name, just need to set ip per current hostname right?

Reply:
fixed